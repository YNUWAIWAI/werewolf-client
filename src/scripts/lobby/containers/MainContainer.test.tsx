import * as React from 'react'
import IntlProviderContainer from './IntlProviderContainer'
import MainContainer from './MainContainer'
import {Provider} from 'react-redux'
import fakeStore from './fakeStore'
import {mount} from 'enzyme'

test('<MainContainer />', () => {
  const store = fakeStore()
  const wrapper = mount(
    <Provider store={store} >
      <IntlProviderContainer>
        <MainContainer />
      </IntlProviderContainer>
    </Provider>
  )

  expect(wrapper.html()).toMatchSnapshot()
})
