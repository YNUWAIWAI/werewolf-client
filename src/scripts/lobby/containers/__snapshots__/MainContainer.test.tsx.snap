// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainContainer /> render 1`] = `"<ul class=\\"lo--menu\\"><li class=\\"lo--menu--item\\"><button class=\\"lo--menu--item--button \\" tabindex=\\"-1\\">Lobby for audiences</button></li><li class=\\"lo--menu--item\\"><button class=\\"lo--menu--item--button \\" tabindex=\\"-1\\">Lobby for robot players</button></li><li class=\\"lo--menu--item\\"><button class=\\"lo--menu--item--button \\" tabindex=\\"-1\\">Lobby for human players</button></li><li class=\\"lo--menu--item\\"><button class=\\"lo--menu--item--button \\" tabindex=\\"-1\\">History</button></li><li class=\\"lo--menu--item\\"><button class=\\"lo--menu--item--button \\" tabindex=\\"-1\\">Settings</button></li><li class=\\"lo--menu--item\\"><button class=\\"lo--menu--item--button \\" tabindex=\\"-1\\">How to play</button></li><li class=\\"lo--menu--item\\"><button class=\\"lo--menu--item--button \\" tabindex=\\"-1\\">Credits</button></li></ul>"`;
